### PR TITLE
DOCSP-46021 - v8.2.1 Release

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-46021
Staging:
- https://deploy-preview-44--docs-entity-framework.netlify.app/#compatibility
- https://deploy-preview-44--docs-entity-framework.netlify.app/whats-new/#what-s-new-in-8.2.1

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by stephmarie17 — APPROVED** [review]
LGTM

**Review by damieng — CHANGES_REQUESTED** [review]
Just one clarification but otherwise LGTM

File: source/whats-new.txt [hunk 1]
```diff
@@ -29,6 +30,26 @@ Learn what's new in:
...
+-------------------
+
+The 8.2.1 provider release includes the following new features:
+
+- Adds support for {+framework+}
+  `shadow properties, <https://learn.microsoft.com/en-us/ef/core/modeling/shadow-properties>`__
+  properties that aren't defined in your .NET entity class but are
+  included in the {+framework-core+} model and mapped to the database.
+
+- Adds support for creating indexes on one or more properties when you call the
```
[the remark this answers is not in the document]
Comment (mongoKart) [reply source/whats-new.txt:45]: Is the use expected to call the `CreateIndex()` method, or do they just have to call `HasIndex()` like you showed in [your example?](https://github.com/mongodb/mongo-efcore-provider/pull/163) I want to make sure I get the details right.

[the remark this answers is not in the document]
Comment (damieng) [reply source/whats-new.txt:45]: Sorry, yes, HasIndex.

File: source/whats-new.txt [hunk 2]
```diff
@@ -42,8 +42,27 @@ The 8.2.1 provider release includes the following new features:
...
+     {
+         mb.Entity<Customer>().HasIndex(c => c.Name);
+         mb.Entity<Order>().HasIndex(o => o.OrderRef).IsUnique();
+         mb.Entity<Address>().HasIndex(o => o.ZipCode, "custom_index_name");
+         mb.Entity<Documents>()
+           .HasIndex(d => d.DocRef)
+           .HasCreateIndexOptions(new CreateIndexOptions { Sparse = true });
+     });
+
+     var result = client.CreateDatabase(context.GetService<IDesignTimeModel>());
```
Comment (damieng) [inline source/whats-new.txt:65]: Let's replace this last line with `context.Database.EnsureCreated();`

**Review by stephmarie17 — APPROVED** [review]
LGTM from the docs writing side

**Pull request merged**